src/context.ts

```diff
@@ -0,0 +1,50 @@
+import React, { createContext, useContext, useRef } from 'react'
+import create, { UseStore } from './index'
```

Comment:
```suggestion
import create, { UseStore } from 'zustand'
```

We need to alias for development. eslintrc does it. tsconfig has to be modified. check how [jotai](https://github.com/pmndrs/jotai) does it.